### ISSUE TITLE
keyboard nav line highlighting not visible in light mode

### ISSUE THREAD

**Issue by markjaquith** [tomasz-tomczyk/crit#506]
In dark mode, you can see which line is highlighted when doing j/k keyboard navigation. In light mode you cannot.

<img width="248" height="61" alt="Image" src="https://github.com/user-attachments/assets/ad468908-6832-4182-ad6e-903c469c86a4" />
<img width="244" height="62" alt="Image" src="https://github.com/user-attachments/assets/ca019478-32fb-4c2b-a7ae-b2f110449cee" />

**Comment by tomasz-tomczyk** [comment]
Thanks for the report! 💚  I'll fix very shortly!

**Comment by tomasz-tomczyk** [comment]
Sorry my crit-web CI is much faster but https://github.com/tomasz-tomczyk/crit/pull/507 will solve it ❤️
